lib/new_relic/agent/instrumentation/grpc/server/chain.rb

```diff
@@ -34,29 +34,29 @@ def run_with_newrelic_trace(*args)
           ::GRPC::RpcDesc.class_eval do
             include NewRelic::Agent::Instrumentation::GRPC::Server
 
-            def handle_request_response_with_newrelic_trace(*args)
-              handle_with_tracing(*args) { handle_request_response_without_newrelic_trace(*args) }
+            def handle_request_response_with_newrelic_trace(active_call, mth, inter_ctx)
+              handle_with_tracing(:request_response, active_call, mth, inter_ctx) { handle_request_response_without_newrelic_trace(active_call, mth, inter_ctx) }
```

Comment:
- For each instrumented gRPC server request handling method, report its name to
  `issue_request_with_tracing`, and then report that name as `grpc_type`
  (thanks, Node.js agent team for the idea)